src/elements/card/src/index.tsx

```diff
@@ -0,0 +1,63 @@
+/** @jsx jsx */
+
+import * as React from 'react'
+import { jsx, useThemeUI } from 'theme-ui'
+
+import { ImgixImage } from '../../imgix-image/src'
```

Comment:
this should use `@uswitch/trustyle.imgix-image`, not a relative path

Reply:
fair